features/web-payment-request.md

```diff
@@ -1,12 +0,0 @@
----
-title: Web Payment Request API
-category: apps
-bugzilla: 1318984
-firefox_status: in-development
-spec_url: https://w3c.github.io/browser-payment-api/
-chrome_ref: 5639348045217792
-ie_ref: Payment Request API
-caniuse_ref: payment-request
```

Comment:
LGTM but note that `caniuse_ref: payment-request` isn't on the remaining record so it should probably be added.